src/views/WellnessCheck/index.js

```diff
@@ -0,0 +1,215 @@
+import Grid from '@material-ui/core/Grid';
+import React, { Component } from 'react';
+import user from '../../services/user';
+import Login from '../Login';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardHeader from '@material-ui/core/CardHeader';
+import Approved from './components/Approved/Approved';
+import Denied from './components/Denied/Denied';
+import wellness from '../../services/wellness';
+import GordonLoader from '../../components/Loader';
+import '../../app.css';
+import { gordonColors } from './../../theme';
+
+export default class WellnessCheck extends Component {
+  constructor(props) {
+    super(props);
+
+    this.logIn = this.logIn.bind(this);
+
+    this.state = {
+      personType: null,
+      network: 'online',
+      currentStatus: 'I am symptomatic',
+      currentUser: null,
+      image: null,
+      loading: true,
+    };
+  }
+
+  async componentDidMount() {
+    try {
+      await this.getUserData();
+      await this.getStatus();
+    } catch (error) {
+      // Do nothing
+    }
+    user.getImage().then(data => {
+      this.setState({ image: data, loading: false });
+    });
+    await this.getStatus();
```

Comment:
Redundant?  Already called in try above.  (Not sure which is the one to keep...)

Reply:
removed one redundant reference